frontend-react/e2e/spec/daily-data-page.spec.ts

```diff
@@ -54,13 +54,14 @@ test.describe("Daily Data page", () => {
             });
 
             test("table has correct headers", async ({ page }) => {
-                await expect(page.getByText(/Report ID/)).toBeAttached();
-                await expect(page.getByText(/Time received/)).toBeAttached();
+                const table = page.locator("table");
+                await expect(table.getByText(/Report ID/)).toBeAttached();
```

Comment:
Can we change this logic to be the following? Otherwise we can only have one "Report ID" on the page for this to work.
You will be able to check for all header labels as well.
`await expect(page.locator(".usa-table th").nth(0)).toHaveText(/Report ID/,);`

Reply:
And check for `Receiver` column